src/coreclr/nativeaot/Runtime/eventpipeinternal.cpp

```diff
@@ -110,26 +133,146 @@ EXTERN_C NATIVEAOT_API void __cdecl RhEventPipeInternal_WriteEventData(
 
 EXTERN_C NATIVEAOT_API UInt32_BOOL __cdecl RhEventPipeInternal_GetSessionInfo(uint64_t sessionID, EventPipeSessionInfo *pSessionInfo)
 {
-    PalDebugBreak();
-    return FALSE;
+    bool retVal = false;
+    if (pSessionInfo != NULL)
+    {
+        EventPipeSession *pSession = EventPipeAdapter::GetSession(sessionID);
+        if (pSession != NULL)
+        {
+            pSessionInfo->StartTimeAsUTCFileTime = EventPipeAdapter::GetSessionStartTime(pSession);
+            pSessionInfo->StartTimeStamp.QuadPart = EventPipeAdapter::GetSessionStartTimestamp(pSession);
```

Comment:
Would it make sense to define the `EventPipeSessionInfo` fields as `int64_t` instead of the Windows-y `FILETIME`/`LARGE_INTEGER`? They are all `long`s on the managed side.